Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -254,26 +256,29 @@ public void SearchValidationsRecursive(DependencyObject depObj)
 
         private void UpdatePrevNextButton()
         {
-
-            if (mWizard.IsLastPage())
+            if (mWizard.IsFirstPage())
```

Comment:
if you will allow collapsing the PrevBtn by exposing the Visability of it so this code whch changing it enabeled or not can remain, anyway the btn will be hidden from UI

Reply:
If we expose the visibility then in case of shared repo PrevBtn is hidden from UI but only we have to disable this so I used bool return type.